packages/snf_mcp/snf_mcp.0.2.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis:
+  "A Model Context Protocol server for web search and content fetching"
+description:
+  "A Model Context Protocol (MCP) server that provides web search and web content fetching capabilities, written in OCaml using the Eio asynchronous runtime. Includes rate limiting and full MCP protocol compatibility."
+maintainer: ["Marcello Seri <marcello.seri@gmail.com>"]
+authors: ["Marcello Seri <marcello.seri@gmail.com>"]
+license: "ISC"
+tags: ["mcp" "fetch" "search" "web" "eio" "ocaml"]
+homepage: "https://github.com/mseri/snf-mcp"
+doc: "https://github.com/mseri/snf-mcp"
+bug-reports: "https://github.com/mseri/snf-mcp/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.17"}
+  "ca-certs"
+  "eio" {>= 0.13}
+  "eio_main"
+  "fmt"
+  "cohttp"
+  "cohttp-eio"
+  "jsonrpc" {>= "1.7.0"}
+  "lambdasoup"
+  "logs"
+  "mirage-crypto-rng" {>= "1.2.0"}
+  "re"
+  "tls-eio"
+  "yojson" {< "3.0.0"}
+  "uri"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
    "@runtest" {with-test & os != "macos"}
```
The sandbox in the CI gets in the way of the server initialization (ca-cert limitations)